app/scripts/views/mixins/settings-panel-mixin.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// helper functions for views with a profile image. Meant to be mixed into views.
+
+define([
```

Comment:
Are settings-panel-mixin and modal-settings-panel-mixin mutually exclusive? They both present a very similar interface. If so, can you make a note at the top of both, and where the use of each is appropriate?